config/routes.rb

```diff
@@ -1,6 +1,10 @@
+require 'resque/server'
+
 Rails.application.routes.draw do
   resources :catalog, param: :druid, only: %i[create update]
   resources :moab_storage, only: %i[index show]
 
+  mount Resque::Server.new, at: '/resque', constraints: { host: /stage-02|prod-03/ }
```

Comment:
So only prod-03 VM has access to the `resque` route?

Reply:
Yep, the dashboard will be set up on stage-02 and prod-03 -- the worker machines.

Reply:
I thought it was:   
- prod-01:   ReST (including Resque dashboard) and M2C, C2M
- prod-02:  CV
- prod-03:  workers

???